include/functions.php

```diff
@@ -201,3 +201,37 @@ function pll_is_plugin_active( string $plugin_name ) {
 
 	return in_array( $plugin_name, $plugins );
 }
+
+/**
+ * Prepares and registers errors.
+ *
+ * Wraps `add_settings_error()` to make its use more consistent.
+ *
+ * @since 3.6
+ *
+ * @param WP_Error $error Error object.
+ * @return void
+ */
+function pll_add_settings_error( WP_Error $error ) {
+	if ( ! $error->has_errors() ) {
+		return;
+	}
+
+	foreach ( $error->get_error_codes() as $error_code ) {
+		// Extract the "error" type.
+		$data = $error->get_error_data( $error_code );
+		$type = empty( $data ) || ! is_string( $data ) ? 'error' : $data;
+
+		$message = wp_kses(
+			$error->get_error_message( $error_code ),
```

Comment:
What about this?
```suggestion
			implode( '<br>', $error->get_error_messages( $error_code ) ),
```
This is done [here](https://github.com/polylang/polylang-pro/blob/fa6667c34f6f0afca29027c0c6a12e80b9f4687c/modules/import-export/export/export-strings-action.php#L161-L162) and [here](https://github.com/polylang/polylang-pro/blob/fa6667c34f6f0afca29027c0c6a12e80b9f4687c/modules/import-export/import/import-action.php#L320-L321).

Reply:
Yes, that's because I was reviewing this other PR that I suggested this ;)